early-bird/src/services/UsersService.js

```diff
@@ -12,4 +13,19 @@ async function GetUserById(id) {
   }
 }
 
-export {GetUserById};
\ No newline at end of file
+export async function Update(password, firstname, lastname) {
+  let path = sessionStorage.getItem("server") + "/api/Users/" + GetUserId();
+  let data = {
+    firstname: firstname,
```

Comment:
what happens if he doesn't want to update his first name/last name?

Reply:
the fields are initially completed using the original first name and last name, if the user doesn't wish to modify those he wont write anything in the first 2 fields.